test/test_binary_ufuncs.py

```diff
@@ -1842,6 +1842,12 @@ def test_nextafter_bfloat16(self, device, dtype):
             actual = torch.nextafter(from_t, to_t).item()
             self.assertEqual(actual, expected, atol=0, rtol=0)
 
+        a = torch.randn(100, device=device, dtype=dtype)
+        b = torch.randn(100, device=device, dtype=dtype)
+        actual = torch.nextafter(a, b)
+        expected = np.nextafter(a.cpu().float().numpy(), b.cpu().float().numpy())
+        self.assertEqual(actual.float(), expected, atol=0.01, rtol=0.01)
```

Comment:
This looks incorrect, (it passes because we do the same operations for numpy such that they both agree)

```python
>>> import torch
>>> import numpy as np
>>> z = torch.zeros(())
>>> z = torch.zeros((), dtype=torch.bfloat16)
>>> o = torch.ones((), dtype=torch.bfloat16)
>>> torch.nextafter(z, o)
tensor(9.1835e-41, dtype=torch.bfloat16)
>>> torch.nextafter(z.float(), o.float())
tensor(1.4013e-45)
>>> np.nextafter(z.float().numpy(), o.float().numpy())
1e-45
```

I think the correct way will be to extend the above golden value test to also check for larger input as here.
```python
for from_v, to_v, expected in cases: 
      # scalar path
     from_t = torch.tensor([from_v], device=device, dtype=dtype) 
     to_t = torch.tensor([to_v], device=device, dtype=dtype) 
     actual = torch.nextafter(from_t, to_t).item()
     self.assertEqual(actual, expected, atol=0, rtol=0) 
    
     # vector path
     from_t = torch.tensor([from_v] * 100, device=device, dtype=dtype) 
     to_t = torch.tensor([to_v] * 100, device=device, dtype=dtype) 
     actual_t = torch.nextafter(from_t, to_t).item()
     expected_t = torch.tensor([expected] * 100, device=device, dtype=dtype) 
     self.assertEqual(actual_t, expected_t, atol=0, rtol=0) 
    

```